src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -146,16 +151,27 @@ public static String generateStepHelp(QuasiDescriptor d){
         StringBuilder mkDesc = new StringBuilder(header(3)).append(" `").append(d.getSymbol()).append("`: ").append(d.real.getDisplayName()).append("\n");
         mkDesc.append("++++\n");
         try{
-            mkDesc.append(generateHelp(new DescribableModel(d.real.clazz), true))
-                .append("\n\n");
-        } catch(Exception ex){
-            ex.printStackTrace();
-            mkDesc.append("<code>").append(ex).append("</code>\n\n");
-        } catch(Error err){
-            err.printStackTrace();
-            mkDesc.append("<code>").append(err).append("</code>\n\n");
+            try {
+                mkDesc.append(generateHelp(new DescribableModel(d.real.clazz), true));
+            } catch (Exception ex) {
+                mkDesc.append(getHelp("help.html", d.real.clazz));
```

Comment:
Not entirely sure where this is; is this in the directory as an alternative?

Reply:
It's already used in the "default" case by describable model, I just had to copy DecribableModel explicitly as we can't always create an instance of it.